latitudesh/lsh
# Feat: Add support to interactive mode to the projects

## What

This PR adds the interactive mode to the projects resource (write actions only). Similarly to the [previous PR](https://github.com/latitudesh/lsh/pull/25), it also covers a refactor on the operations, to simplify the generated code.

## How to test

Run the build and then test the following commands:

- `./lsh projects create`
- [ ] It should prompt the attributes to create a project
- [ ] When all the attributes are filled correctly, the project should be created and returned in the response
---
- `./lsh projects update`
- [ ] It should first prompt the ID/Slug from the project, and then the attributes you want to update (blank attributes will be skipped)
---
- `./lsh projects destroy`
- [ ] Should prompt the project ID
---
You should also be able to mix interactive mode with flags, for example:

`./lsh projects create --provisioning_type on_demand` (it should prompt all the fields except the provisioning_type)
